content/module-reference/utility-modules/darkroom/high-quality-processing.md

```diff
@@ -0,0 +1,29 @@
+---
+title: high quality processing
+id: hq-processing
+applicable-version: 4.8
+tags:
+view: darkroom
+---
+
+Click the ![high quality processing icon](./high-quality-processing/high-quality-processing-icon.png#icon) icon to activate high quality processing in the pixelpipe.
+
+Darktable pixelpipes process the modules in slightly different ways, the thumbnail pipe as an example is optimized for speed.
```

Comment:
I generally try to avoid starting sentences with the word "darktable" so that I can maintain the idiosyncratic lowercase form without breaking language. If you can't think of a way to reword the sentence it's fine like this.